.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js: 6
+group: bluezone
+install:
+  - npm install
+script:
+  - npm run ci-check
+before_deploy:
```

Comment:
Want to omit this block and I'll add it in with the TravisCI update for build/deploy/publishing?